File: app/src/main/java/com/sorcererxw/matthiasheiderichphotography/util/ApplicationUtil.java
package com.sorcererxw.matthiasheiderichphotography.util;

import android.content.Context;
import android.content.pm.PackageManager;

/**
 * @description:
 * @author: Sorcerer
 * @date: 2016/10/14
 */

public class ApplicationUtil {
    public static boolean isApplicationInstalled(Context context,String packageName){
        try {
            context.getPackageManager().getApplicationInfo(packageName, 0);
            return true;
        }
        catch (PackageManager.NameNotFoundException e) {
            return false;
        }
    }
}

File: app/src/main/java/com/sorcererxw/matthiasheiderichphotography/util/ApplicationUtil.kt
package com.sorcererxw.matthiasheiderichphotography.util

import android.content.Context
import android.content.pm.PackageManager

/**
 * @description:
 * @author: Sorcerer
 * @date: 2016/10/14
 */

object ApplicationUtil {
    fun isApplicationInstalled(context: Context, packageName: String): Boolean {
        return try {
            context.packageManager.getApplicationInfo(packageName, 0)
            true
        } catch (e: PackageManager.NameNotFoundException) {
            false
        }
    }
}
